rpmUtils/miscutils.py

```diff
@@ -72,7 +72,7 @@ def checkSig(ts, package):
     try:
         hdr = ts.hdrFromFdno(fdno)
     except rpm.error, e:
-        if str(e) == "public key not availaiable":
```

Comment:
To expand on megaumi's comment, this is presumably for an old typo in rpm ... we even check the "correct" string below.